exporter/stackdriverexporter/go.mod

```diff
@@ -4,11 +4,13 @@ go 1.14
 
 require (
 	contrib.go.opencensus.io/exporter/stackdriver v0.13.1
+	github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace v0.2.1-0.20200710010320-8556f4b5bc41
```

Comment:
The latest version is from Jul 29, 2020 (https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/commit/2752da7eaab799cbc680f91181a6af43866409b0), you need to update to it to include your changes on that side?